packages/env-tests/package.json

```diff
@@ -6,6 +6,7 @@
   "license": "MIT",
   "dependencies": {
     "@celo/contractkit": "1.0.0-dev",
+    "@celo/celotool": "1.0.0",
```

Comment:
This is exactly what I tried to avoid. Celotool is a very heavy package with a lot of dependencies